### ISSUE TITLE
[Feature Request] Buffer destination directory

### ISSUE THREAD

**Issue by puppetmaster886** [swar/Swar-Chia-Plot-Manager#962]
Add a buffer destination directory to the jobs.
Most of the final destination drives (the one that we will use for farming) are very slow, then if we set them as destination the copy phase will take too much time.
The idea is to copy the plots a much faster drive (like an SSD) to decrease the copy time and allow to start earlier with a new job, and then move the plot from that drive to the final destination for farming.

**Comment by creamsoup** [comment]
i am doing this with simple bash script scheduled with systemd timer. simple enough. thus, #452 can allow you to do that much easily.

**Comment by g0ze21** [comment]
Could you share it?

**Comment by tob-pas** [comment]
I do this also on windows with a powershell script.
so I can start earlier with the next job, but also I have no need for high staggers, because powershell script copies only one plot in time

**Comment by g0ze21** [comment]
> I do this also on windows with a powershell script.
> so I can start earlier with the next job, but also I have no need for high staggers, because powershell script copies only one plot in time

Please share your script bro.

**Comment by tob-pas** [comment]
thats the code of my powershell script. name it something.PS1
I created a scheduled task which runs every 5 minutes.

[move_plots.ps1.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6564196/move_plots.ps1.txt)

**Comment by maxcom99** [comment]
> thats the code of my powershell script. name it something.PS1
> I created a scheduled task which runs every 5 minutes.
> 
> [move_plots.ps1.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6564196/move_plots.ps1.txt)

Do you have a repo for this?

**Comment by tob-pas** [comment]
> > thats the code of my powershell script. name it something.PS1
> > I created a scheduled task which runs every 5 minutes.
> > [move_plots.ps1.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6564196/move_plots.ps1.txt)
> 
> Do you have a repo for this?

no, there is no repo. thats a five minute script for me (you see it also that there is my local C-drive with path directly in the robocopy script ;-) )
